ci/run-integration-tests.sh

```diff
@@ -2,11 +2,11 @@
 
 set -o errexit -o pipefail -o nounset
 
-if [[ ! -d examples || ! -d CMakeFiles ]]; then
+if [[ ! -d CMakeFiles ]]; then
     echo "Warning: This script should be run from within a build directory" 1>&2
 fi
 
-if [[ $# -le 1 ]]; then
+if [[ $# -le 2 ]]; then
```

Comment:
Why 2? `$#` does not include `$0`!

Reply:
Exactly, otherwise it would have to be `-lt 3` (I want to see at least one node-count)

Reply:
Yes but this is `-le` not `-lt`

Reply:
Right!